### PR TITLE
Css brochure form

### PR DESCRIPTION
**Pull request by bug-smith**
completed.

Screenshot 2023-09-07 at 6 34 19 PM

### CODE REVIEW

File: css-brochure-form/index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    <title>CSS Challenge - Brochure Form</title>
+    <link rel="stylesheet" href="fonts.css" />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div>
+      <form>
+        <h1 class="header">Learn More</h1>
+        <p class="email">Company Email</p>
```
Comment (shawnkost) [inline css-brochure-form/index.html]: This should be a `label` and should be linked to your `input`

File: css-brochure-form/index.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        <input
+          type="email"
+          id="your-email"
+          name="email"
+          placeholder="sally@company.co" />
+        <p class="name">Name</p>
```
Comment (shawnkost) [inline css-brochure-form/index.html]: This should be a `label` and should be linked to your `input`

File: css-brochure-form/index.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        <input type="text" id="name" name="name" placeholder="Sally Smith" />
+        <p class="hiring-location">Where are you hiring?</p>
```
Comment (shawnkost) [inline css-brochure-form/index.html]: This should be a `label` and should be linked to your `input`

File: css-brochure-form/index.html [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        <input
+          type="text"
+          id="hiring-location"
+          name="hiring-location"
+          placeholder="City, County" />
+        <div class="button">
+          <p class="how-many">
+            How many engineers are you hiring in the next 6 months?
+          </p>
```
Comment (shawnkost) [inline css-brochure-form/index.html]: This should be a `label` and should be linked to your `select`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Few changes here!

**Comment by bug-smith** [comment]
updated to be labels + block, that are for the text/select

Screenshot 2023-09-08 at 10 07 01 AM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**